<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDeviceWidget</class>
 <widget class="QWidget" name="SearchDeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchDeviceWidget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>13</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>选择板子：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>43</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>元件类型：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_board">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_type">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_search">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_device">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>100</y>
     <width>200</width>
     <height>420</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
